p2p/discover/table.go

```diff
@@ -115,6 +117,11 @@ func newTable(t transport, db *enode.DB, bootnodes []*enode.Node, log log.Logger
 		log:        log,
 		enrFilter:  filter,
 	}
+	if bootnode {
```

Comment:
Just a personal opinion, I would avoid using the else statement and assign `bucketSize` by default to the `tab.bucketSize` and change it in the if statement